draft-ietf-quic-applicability.md

```diff
@@ -478,6 +481,13 @@ As long as the new path is not validated only probing packets can be sent.
 However, the probing packets can be used measure path characteristics as
 input for the switching decision or the congestion controller on the new path.
 
+Only the client can actively migrate, however, servers can indicate during the
+handshake that they prefer to transfer the connection to a different address
+after the handshake, e.g. to move from an address that is shared by multiple
+servers to an unicast address. The server can provide an IPv4 and an IPv6
```

Comment:
```suggestion
after the handshake, e.g., to move from an anycast address to a unicast
address. The server can provide an IPv4 and an IPv6
```

Though it might be better to instead say.

```suggestion
after the handshake, e.g. to move from an address that is shared by multiple
servers to an address that is unique to the server instance. The server can
provide an IPv4 and an IPv6
```

...as that allows for the possibility of both anycast -> unicast transitions and load balanced to non-load balanced transitions both.